server/jvm/gradle.properties

```diff
@@ -2,9 +2,6 @@ kotlin.code.style=official
 org.gradle.jvmargs=-Xmx6g -Xss512k -XX:+HeapDumpOnOutOfMemoryError -XX:+UseG1GC -XX:+UseStringDeduplication -XX:ReservedCodeCacheSize=512m -Dkotlin.daemon.jvm.options=-Xmx2g -Dfile.encoding=UTF-8
 bundleGeneratedClasses=true
 genesisVersion={{versions.GSF}}
-authVersion={{versions.Auth}}
-deployPluginVersion={{versions.GSF}}
-genesisArtifactoryPath=https://genesisglobal.jfrog.io/genesisglobal/libs-release-client
```

Comment:
Why have these been removed?

Reply:
I've been using the simple app seed as a guide which doesn't add auth by default. As for `deployPluginVersion` it's not needed anymore since that's managed by settings plugin.  The simple app seed doesn't apply the artifactory plugin so also got rid of it, `genesisArtifactoryPath`  was only used by that plugin

Reply:
Agree we should then remove the deployPluginVersion as we are managing that in the Settings plugin, but I think we should keep everything else as potentially docs/training materials etc are dependent on these things.